xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -1165,6 +1169,25 @@ bool CDVDDemuxFFmpeg::SeekTime(double time, bool backwards, double* startpts)
 
   if (m_checkvideo)
   {
+    int msCount = 1000;
+
+    while (!IsVideoReady() && msCount > 0)
+    {
+      DemuxPacket *pkt = Read();
+      if (pkt)
```

Comment:
omitting brackets on one condition and using them on the other turned out to be bad coding style. there were cases where devs added lines to the block without adding the brackets.
not that I care, just mentioning it

Reply:
Fair point. Will change when I add the timer.